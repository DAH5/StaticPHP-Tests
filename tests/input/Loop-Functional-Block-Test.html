--- loop( dir = "tests/input/loop" ) ---
<p><b>--- loop.title ---</b></p>
<p>--- loop.description ---</p>
<p><a href="--- loop.uri ---">Read More</a></p>
--- endloop ---