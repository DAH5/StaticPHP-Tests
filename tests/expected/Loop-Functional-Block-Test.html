
<p><b>Hello, world!</b></p>
<p>This is a hello world file.</p>
<p><a href="/loop/hello-world.html">Read More</a></p>

<p><b>Shopping List</b></p>
<p>NOTE TO SELF: What to get when grocery shopping.</p>
<p><a href="/loop/shopping-list.html">Read More</a></p>
